### PR TITLE
update(doozle): improvements

### PR DESCRIPTION
**Pull request by falkheiland**
- added option to enable actions (start, restart, stop containers)
- changed authentication configuration introduced in https://github.com/runtipi/runtipi-appstore/pull/3070
- added option to not use authentication
- added instructions to create password hash to the app description (from the originally provided gist link)

if you already have configured your authentication, it will continue to work, when authentication is enabled.

tagging https://github.com/Hunam6 

## Summary by CodeRabbit

- **New Features**
  - Introduced a new "Auth Provider" field for improved authentication options.
  - Added "Enable Actions" field to control container actions directly from the UI.

- **Bug Fixes**
  - Updated user credentials configuration for better authentication handling.

- **Chores**
  - Improved the flexibility of configuration options in the Docker Compose setup.

### CODE REVIEW

File: apps/dozzle/data/users.yml.template [hunk 1]
```diff
@@ -1,4 +1,4 @@
 users:
-  {{DOZZLE_USERNAME}}:
-    name: {{DOZZLE_USERNAME}}
-    password: "{{DOZZLE_PASSWORD_HASH}}"
\ No newline at end of file
+  admin:
+    name: admin
+    password: "5e884898da28047151d0e56f8dc6292773603d0d6aabbdd62a11ef721d1542d8"
```
[the remark this answers is not in the document]
Comment (falkheiland) [reply apps/dozzle/data/users.yml.template:4]: this is a default standard password. instructions to change are included in the description

[the remark this answers is not in the document]
Comment (falkheiland) [reply apps/dozzle/data/users.yml.template:4]: won't fix. designed as intended.

**Comment by JigSawFr** [comment]
@falkheiland please restore original app version and add the two missing fields in config.json (created_at, updated_at)
Thanks !

**Comment by falkheiland** [comment]
> @falkheiland please restore original app version and add the two missing fields in config.json (created_at, updated_at) Thanks !

done

**Comment by JigSawFr** [comment]
thanks @falkheiland

**Pull request merged**